docs/release-notes/eclair-v0.10.0.md

```diff
@@ -0,0 +1,276 @@
+
+There is currently no way to automatically add funds to channels that are being opened to your node, as deciding whether to do so or not really depends on each node operator's peering strategy.
+We have however created a [sample plugin](https://github.com/ACINQ/eclair-plugins/tree/master/channel-funding) that node operators can fork to implement their own strategy for contributing to inbound channels.
+
+### Update minimal version of Bitcoin Core
+
+With this release, eclair requires using Bitcoin Core 24.1.
+Newer versions of Bitcoin Core may be used, but haven't been extensively tested.
+
+### Bolt12
+
+Eclair supports the latest state of the [Bolt 12 specification](https://github.com/lightning/bolts/pull/798).
+The specification may still change at that point, so support is experimental, but you may try it using:
+
+- the `payoffer` RPC to pay a Bolt 12 offer
+- the [tip jar plugin](https://github.com/ACINQ/eclair-plugins/tree/master/bolt12-tip-jar) to generate Bolt 12 offers
+
+This may also be compatible with other implementations, but there's no guarantee at that point.
+
+### Use priority instead of block target for feerates
+
+Eclair now uses a `slow`/`medium`/`fast` notation for feerates (in the style of mempool.space),
+instead of block targets. Only the funding and closing priorities can be configured, the feerate
+for commitment transactions is managed by eclair, so is the fee bumping for htlcs in force close
+scenarii. Note that even in a force close scenario, when an output is only spendable by eclair, then
+the normal closing priority is used.
+
+Default setting is `medium` for both funding and closing. Node operators may configure their values like so:
+
+```eclair.conf
+eclair.on-chain-fees.confirmation-priority {
+    funding = fast
+    closing = slow
+}
+```
+
+This configuration section replaces the previous `eclair.on-chain-fees.target-blocks` section.
+
+### Add configurable maximum anchor fee
+
+Whenever an anchor outputs channel force-closes, we regularly bump the fees of the commitment transaction to get it to confirm.
+We previously ensured that the fees paid couldn't exceed 5% of our channel balance, but that may already be extremely high for large channels.
+Without package relay, our anchor transactions may not propagate to miners, and eclair may end up bumping the fees more than the actual feerate, because it cannot know why the transaction isn't confirming.
+
+We introduced a new parameter to control the maximum fee that will be paid during fee-bumping, that node operators may configure:
+
+```eclair.conf
+// maximum amount of fees we will pay to bump an anchor output when we have no HTLC at risk
+eclair.on-chain-fees.anchor-without-htlcs-max-fee-satoshis = 10000
+```
+
+We also limit the feerate used to be at most the same order of magnitude of `fatest` feerate provided by our fee estimator.
```

Comment:
```suggestion
We also limit the feerate used to be at most of the same order of magnitude as the `fastest` feerate provided by our fee estimator.
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/2831/commits/8772d9657e2aca4a8cad8a31d5d4dcb1fe6a8b8b